src/java/edu/slu/tpen/servlet/CanvasServlet.java

```diff
@@ -100,76 +103,88 @@ public String getServletInfo() {
         build the JSON representation of a canvas and return it.  It will not know about the project, so otherContent will contains all annotation lists this canvas
         has across all projects.  It will ignore all user checks so as to be open.  
     */
-    private JSONObject buildPage(Folio f) throws SQLException, IOException {
-      Integer msID = f.getMSID();
-      String msID_str = msID.toString();
-      String canvasID = getRbTok("SERVERURL")+"canvas/"+f.getFolioNumber();  
-      String[] otherContent;
-      FolioDims pageDim = new FolioDims(f.getFolioNumber(), true);
-      Dimension storedDims = null;
-      int canvasWidth = 0;
-      int canvasHeight = 0;
-      if (pageDim.getImageHeight() <= 0) { //There was no foliodim entry
-          storedDims = getImageDimension(f.getFolioNumber());
-         if(null == storedDims || storedDims.height <=0){ //There was no imagecache entry, or there was a bad one we can't use
-            storedDims = f.getImageDimension(); //Resolve the image headers and get the image dimensions
-         }
-      }
-      //Logger.getLogger(CanvasServlet.class.getName()).log(Level.INFO, "pageDim={0}", pageDim);
-      JSONObject result = new JSONObject();
-      result.element("@id", canvasID);
-      result.element("@type", "sc:Canvas");
-      result.element("label", f.getPageName());
-      
-      JSONArray images = new JSONArray();
-      JSONObject imageAnnot = new JSONObject();
-      imageAnnot.element("@type", "oa:Annotation");
-      imageAnnot.element("motivation", "sc:painting");
-      String imageURL = f.getImageURL();
-      if (imageURL.startsWith("/")) {
-        imageURL = String.format("%spageImage?folio=%s",getRbTok("SERVERURL"), f.getFolioNumber());
-      }
-      Map<String, Object> imageResource_map = buildQuickMap("@id", imageURL, "@type", "dctypes:Image", "format", "image/jpeg");
-      JSONObject imageResource = fromObject(imageResource_map);
-      imageResource.element("height",0 ); 
-      imageResource.element("width",0 ); 
-      
-      if (storedDims != null) {//Then we were able to resolve image headers and we have good values to run this code block
-            if(storedDims.height > 0){//The image header resolved to 0, so actually we have bad values.
-                if(pageDim.getImageHeight() <= 0){ //There was no foliodim entry, so make one.
-                    //generate canvas values for foliodim
-                    canvasHeight = 1000;
-                    canvasWidth = storedDims.width * canvasHeight / storedDims.height; 
-                    createFolioDimsRecord(storedDims.width, storedDims.height, canvasWidth, canvasHeight, f.getFolioNumber());
-                }
+     private JSONObject buildPage(int projID, Folio f) throws SQLException, IOException {
```

Comment:
This is not the right build page.  You are not getting the Annotation lines.  You need to use the same buildPage that JSONLDExporter.java is using.